doc/en/warnings.rst

```diff
@@ -33,6 +33,8 @@ Running pytest now produces this output::
       $REGENDOC_TMPDIR/test_show_warnings.py:4: UserWarning: api v1, should use functions from v2
         warnings.warn(UserWarning("api v1, should use functions from v2"))
 
+    -- Docs: https://docs.pytest.org/en/latest/warnings.html
+    ========================= warnings summary (final) =========================
     -- Docs: https://docs.pytest.org/en/latest/warnings.html
```

Comment:
hmm, this is a little odd -- printing twice now?

Reply:
the twice is not a mistake, but it seems a set of conditionals is misaligned

Reply:
Ok, will push your suggested fix here.